### PR TITLE
fix pep8 issues in psf subpackage

### PR DESCRIPTION
**Pull request by mirca**

### CODE REVIEW

File: photutils/psf/tests/test_core.py [hunk 1]
```diff
@@ -98,9 +105,12 @@ def test_psf_adapter(moffimg, prepkwargs, tols):
...
 
     # a close-but-wrong "guessed Moffat"
-    guess_moffat = Moffat2D(x_0=.1, y_0=-.05, gamma=1.01, amplitude=mof.amplitude*1.01, alpha=4.79)
+    guess_moffat = Moffat2D(x_0=.1, y_0=-.05, gamma=1.01,
+                            amplitude=mof.amplitude*1.01, alpha=4.79)
     if prepkwargs['renormalize_psf']:
-        guess_moffat.amplitude = 5.  # definitely very wrong, so this ensures the re-normalization stuff works
+        guess_moffat.amplitude = 5.
+        # definitely very wrong, so this ensures the re-normalization
+        # stuff works
```
Comment (larrybradley) [inline photutils/psf/tests/test_core.py]: This comment should go before the line `guess_moffat.amplitude = 5.`, not after it.

File: photutils/psf/tests/test_core.py [hunk 2]
```diff
@@ -98,9 +105,12 @@ def test_psf_adapter(moffimg, prepkwargs, tols):
     f = LevMarLSQFitter()
 
     # a close-but-wrong "guessed Moffat"
-    guess_moffat = Moffat2D(x_0=.1, y_0=-.05, gamma=1.01, amplitude=mof.amplitude*1.01, alpha=4.79)
+    guess_moffat = Moffat2D(x_0=.1, y_0=-.05, gamma=1.01,
+                            amplitude=mof.amplitude*1.01, alpha=4.79)
     if prepkwargs['renormalize_psf']:
-        guess_moffat.amplitude = 5.  # definitely very wrong, so this ensures the re-normalization stuff works
+        guess_moffat.amplitude = 5.
+        # definitely very wrong, so this ensures the re-normalization
```
Comment (bsipocz) [inline photutils/psf/tests/test_core.py]: I would move this comment two lines up (at least I always assume the comment is either inline or a comments the code below it)

**Comment by bsipocz** [comment]
Other than my tiny comment this looks good. You can add ``[skip ci]`` to the commit message, so it won't trigger the tests again.

**Comment by larrybradley** [comment]
Thanks, @mirca!

**Pull request merged**